### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by julianrrr**
### Changes
Added a Spanish translation files, F-Droid/Play Store metadata and edited the credit file according to the instructions.

### Testing
- [X] Tested on a physical device

### CODE REVIEW

**Comment by ashutoshgngwr** [comment]
Ignore if 732.3 fails. It wasn't supposed to run.

File: fastlane/metadata/android/es-ES/short_description.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Relájate, mejora el enfoque y aumenta la productividad con un mínimo ruido de fondo.
```
Comment (ashutoshgngwr) [inline fastlane/metadata/android/es-ES/short_description.txt:1]: We need to keep it under 80 characters. Its a restriction from the Play Store.

Comment (julianrrr) [reply fastlane/metadata/android/es-ES/short_description.txt]: Done, also I adjusted two translation values. :wink:

**Comment by ashutoshgngwr** [comment]
Wow. Sometimes my actions do lead to a mess. @julianrrr 

If you see this message, I'll take it from here. Merging master into your forks 0.7.x caused some more issues I think.
https://github.com/ashutoshgngwr/noice/blob/9b16581b0dcd2ff500628f783e59e743c2251cc8/app/src/main/java/com/github/ashutoshgngwr/noice/fragment/AboutFragment.kt#L153

Please leave it be for the time being. I see you have committed the change I requested. I'll fix this from my end. Sorry for the mess.

**Comment by julianrrr** [comment]
> Wow. Sometimes my actions do lead to a mess. @julianrrr
> 
> If you see this message, I'll take it from here. Merging master into your forks 0.7.x caused some more issues I think.
> 
> https://github.com/ashutoshgngwr/noice/blob/9b16581b0dcd2ff500628f783e59e743c2251cc8/app/src/main/java/com/github/ashutoshgngwr/noice/fragment/AboutFragment.kt#L153
> 
> Please leave it be for the time being. I see you have committed the change I requested. I'll fix this from my end. Sorry for the mess.
> 
> **Edit**: Please make sure that you have "Allow edits from Maintainers" option enabled on the PR.

Ok, the option is enable. Tell me if anymore need to be adjusted

**Comment by ashutoshgngwr** [comment]
@julianrrr can you please verify if everything is as you intended it to be?

**Comment by ashutoshgngwr** [comment]
This looks fine to me. I'll go ahead and merge this. Still let me know if anything is not okay.

**Pull request merged**